pkg/builder/builder.go

```diff
@@ -472,5 +456,15 @@ func (payloadConfig *PayloadConfig) GeneratePayload(filename string, payload []b
 		}
 	}
 
-	return goBinFile, nil, nil
+	return binFile, nil, nil
```

Comment:
I think this break the api mode, thats why i decided to return a byte array

Reply:
will defer to you, haven't done enough testing of this

Reply:
So if i renember well, you changed to return the path instead of the content of the binary ?

Reply:
- [ ] See if the changes impact the api mode.

Reply:
purely anecdotal and no comprehensive, api/server mode works as is.

Reply:
Cool ! Thanks doe the test